### DISCUSSION TITLE
How to fix properly TS7016: Could not find a declaration file for module 'ziggy-js'.

### DISCUSSION THREAD

**Discussion by Nicolas-Her** [tighten/ziggy#716] [Q&A] [answered]
### Ziggy version
2.0.3

### Laravel version
10.45.0

### Additional info
TypeScript: 5.3.3
Vite: 5.1.2
Vue: 3.4.19

### Description
Hi when i run 
`vue-tsc --noEmit -p tsconfig.app.json --composite false`

i have many error 
```
TS7016: Could not find a declaration file for module 'ziggy-js'. '/app/node_modules/ziggy-js/dist/index.js' implicitly has an 'any' type.
  There are types at '/app/node_modules/ziggy-js/src/js/index.d.ts', but this result could not be resolved when respecting package.json "exports". The 'ziggy-js' library may need to update its package.json or typings.
```

from file who as 
`import { route } from 'ziggy-js'`

How to fix them properly ?

**Comment by rafalosinski** [comment]
similar situation after latest composer install and npm install

> tsc && vite build && vite build --ssr && vite build && vite build --ssr --ssr

```
error TS2688: Cannot find type definition file for 'ziggy-js'.
  The file is in the program because:
    Entry point for implicit type library 'ziggy-js'
```

**Comment by bakerkretzmar** [comment, marked as answer]
@Nicolas-Her can you upgrade to v2.0.5 and see if that fixes it?
